setup.py

```diff
@@ -275,6 +275,7 @@ def _setup_entry_points() -> Dict:
             "deepsparse.server=deepsparse.server.cli:main",
             "deepsparse.object_detection.annotate=deepsparse.yolo.annotate:main",
             "deepsparse.yolov8.annotate=deepsparse.yolov8.annotate:main",
+            "deepsparse.yolov8.eval=deepsparse.yolov8.validation:main",
```

Comment:
@dbogunowicz thoughts on changing to `deepsparse.ultralytics.val`?
1. ultralytics will be consistent with sparseml (and is the package name). motivation is that ultralytics actually supports yolov3/yolov5 as well, so its not limited to yolov8
2. val will be consistent with other naming

Reply:
I think that the big plan is just to have an `object_detection` endpoint, that will use by default the best possible architecture/method/framework. That means that once we get the results from the research on yolov8 (and those results probably should be superior to yolov5), we will pick the best model on the accuracy/efficiency curve and this endpoint will be calling it.